namespaces/live-1.cloud-platform.service.justice.gov.uk/opg-gt-test-prototype/resources/github-repo.tf

```diff
@@ -0,0 +1,54 @@
+data "github_team" "default" {
+  slug = "gt-test-team"
+}
```

Comment:
Ran a manual plan and it appears to be failing on this value with the following error:
```
Error: GET https://api.github.com/orgs//teams/gt-test-team: 404 Not Found []

  on github-repo.tf line 1, in data "github_team" "default":
   1: data "github_team" "default" {
```
I noticed the team is marked "secret", is there a "public" team you can specify?

Reply:
For example, you're in the `opg-webops` team with 10 other people. Is it okay for the other 9 to have access?

Reply:
Hi Jason, I've made that team public so hopefully should be able to run now 🤞🏽 

I remember thinking at the time "I wonder if this will cause a problem" but forgot to note it on the PR 🤦🏽‍♀️

Reply:
I did a force-push to restart the job but looks like it's still not happy :(

Reply:
It looks like it might be a wider issue (not related to this PR). I'll do everything manually from here on out. Just re-checking...

Reply:
Cool, that comes up clean. I'll approve.